common/lib/transport/connectionmanager.js

```diff
@@ -551,6 +551,11 @@ var ConnectionManager = (function() {
 				state = 'disconnected';
 			}
 			this.notifyState({state: state, error: error});
+		} else if(wasActive) {
```

Comment:
First case applies also when wasActive is false (when the transport being deactivated was pending and there is no active and no further pending transports), that change would break that case